cfgov/unprocessed/css/organisms/form.less

```diff
@@ -51,8 +51,13 @@
     margin-bottom: unit( @grid_gutter-width * 2 / @base-font-size-px, em);
 
     .respond-to-max(@bp-sm-max, {
-        .btn {
+        .a-btn {
             width: 100%;
```

Comment:
Hmm looking at the code here they look like they should be the same, but they don't behave the same when I switch them out.